sstables/sstables_manager.hh

```diff
@@ -128,6 +129,11 @@ public:
         return _storage->get_endpoint_client(std::move(endpoint));
     }
 
+    bool has_endpoint_client(sstring endpoint) const {
```

Comment:
IMHO, this name is kind of confusing. we are not checking if we have one or more clients for a given endpoint, instead we check if the endpoint is configured.

probably we can rename this method to `is_known_endpoint()`?

Reply:
> IMHO, this name is kind of confusing. we are not checking if we have one or more clients for a given endpoint, instead we check if the endpoint is configured.

Isn't it the same? In this particular place I mean

Reply:
Renamed anyway